app/templating/summary/question.py

```diff
@@ -67,11 +78,11 @@ def _build_checkbox_answers(cls, answer, answer_schema):
         multiple_answers = []
         for option in answer_schema['options']:
             if option['value'] in answer:
-                if option['value'] == 'other':
+                if option['value'].lower() == 'other':
                     summary_option_display_value = cls._get_checkbox_other_display_value(answer, answer_schema, option)
```

Comment:
If we don't need the `_get_checkbox_other_display_value` method we should remove it and just use the label